.buildkite/pipeline.yml

```diff
@@ -26,13 +64,34 @@ steps:
     agents:
       - queue=elastic
 
-  - name: ':skull: Mutest'
-    command: build/run 2.3.3 'rake metrics:mutest'
-    timeout_in_minutes: 10
+  - name: ':rubocop: Rubocop'
```

Comment:
We probably only need to run rubocop on `2.2.7`, don't you think? We might get slight conflicts between the versions and really only need to adhere to one.

Unless maybe for adding string literal comments, i guess?